test/test_util_pipe.py

```diff
@@ -78,19 +78,25 @@ def test_pipe_protocol(self):
             pipe_writer_handler = pipe.Pipe(self.NAME, self.VERSION)
 
         __pool = concurrent.futures.ThreadPoolExecutor()
+        for count in range(100):
+            for message in to_send:
+                for iteration in range(20):
```

Comment:
Are those repetitions really needed?

Reply:
The most nested loop is indeed unneeded. The pipe should sent/receive the data at the first try.

Iterating over `to_send` is needed from obvious reasons.

Repeting the case allows us to catch some edge cases that were difficult to find on GithubActions, so I'd keep this 100.